examples/gno.land/p/gnoland/commondao/proposal.gno

```diff
@@ -8,20 +8,18 @@ import (
 	"gno.land/p/demo/avl"
 )
 
-// DefaultQuorum defines the default quorum required to tally proposal votes.
-const DefaultQuorum = 0.34 // 34%
```

Comment:
All quorum functionality has been removed to allow devs to check it within their `ProposalDefinition` implementation is demded necessary